.gitlab/source_test/ebpf.yml

```diff
@@ -88,16 +77,53 @@ tests_ebpf_x64:
   tags: ["runner:main"]
   variables:
     ARCH: amd64
+    TASK_ARCH: x64
 
 tests_ebpf_arm64:
   extends: .tests_linux_ebpf
   image: 486234852809.dkr.ecr.us-east-1.amazonaws.com/ci/datadog-agent-buildimages/system-probe_arm64:$DATADOG_AGENT_SYSPROBE_BUILDIMAGES
   tags: ["runner:docker-arm", "platform:arm64"]
   variables:
     ARCH: arm64
+    TASK_ARCH: arm64
+
+.prepare_ebpf_tests:
+  stage: source_test
+  needs: ["go_deps"]
+  artifacts:
+    when: always
+    paths:
+      - $CI_PROJECT_DIR/.tmp/binary-ebpf
+      - $DD_AGENT_TESTING_DIR/site-cookbooks/dd-security-agent-check/files
+      - $DD_AGENT_TESTING_DIR/site-cookbooks/dd-system-probe-check/files
+  before_script:
+    - !reference [.retrieve_linux_go_deps]
+    - mkdir -p $CI_PROJECT_DIR/.tmp/binary-ebpf/co-re
+    - inv -e install-tools
+    - !reference [.retrieve_sysprobe_deps]
+  script:
+    - !reference [.build_sysprobe_artifacts]
+    - invoke -e security-agent.kitchen-prepare --skip-linters
+    - cp /tmp/clang-bpf $DD_AGENT_TESTING_DIR/site-cookbooks/dd-system-probe-check/files/clang-bpf
+    - cp /tmp/llc-bpf $DD_AGENT_TESTING_DIR/site-cookbooks/dd-system-probe-check/files/llc-bpf
+
+prepare_ebpf_tests_arm64:
```

Comment:
Nit: would it make sense to make these job names more specific? Right now, their name make them sound like prep step for the `tests_ebpf_*` unit test jobs, but they are prep steps jobs in the `functional_test` stage.
Something like `prepare_ebpf_functional_tests_arm64` / `prepare_ebpf_functional_tests_x64`?